--- a/Makefile	2021-05-23 09:30:50.000000000 +0800
+++ b/Makefile	2021-05-27 22:50:58.000000000 +0800
@@ -531,7 +531,7 @@ endif
 deps/lua/src/liblua.a:
 	$(MAKE) -C deps/lua/src CC="$(CC) $(LUADEFINES)" CXX="$(CXX)" MYCFLAGS="$(LUA_MYCFLAGS) -w $(fpic)" MYLDFLAGS="$(LDFLAGS) $(fpic)" SYSCFLAGS="$(LUA_SYSCFLAGS) $(fpic)" liblua.a
 deps/luajit/src/libluajit.a:
-	$(MAKE) -C deps/luajit/src CC="$(CC)" CXX="$(CXX)" BUILDMODE=static CFLAGS="$(LUA_MYCFLAGS) -w $(fpic)" Q= LDFLAGS="$(LDFLAGS) $(fpic)" libluajit.a
+	$(MAKE) -C deps/luajit/src HOST_CC="$(CC)" HOST_CXX="$(CC)" BUILDMODE=static CFLAGS="$(LUA_MYCFLAGS) -w $(fpic)" Q= LDFLAGS="$(LDFLAGS) $(fpic)" libluajit.a
 
 $(INTDIR)/%.o: %.c
 	@mkdir -p $(dir $@)

